task/docker-build/0.1/docker-build.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: docker-build
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: docker, build-image
+    tekton.dev/displayName: docker
+spec:
```

Comment:
Can we have description added under `spec` here as :

  ```yaml
  metadata:
    labels:
      app.kubernetes.io/version: "0.1"
    annotations:
      tekton.dev/pipelines.minVersion: "0.12.1"   
      tekton.dev/tags: <tags associated>
      tekton.dev/displayName: An Optional name for the task 

  spec:
     description: >-
       One line summary of the task. 

       Description of the task 
  ```

See : #389

Reply:
@PuneetPunamiya Sorry, I thought I fixed that. I saw that you mentioned it before. I just added it.